website/docs/providers/entra/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Entra ID provider
+---
+
+<span class="badge badge--primary">Enterprise</span>
+
+---
+
+With the Microsoft Entra ID provider, authentik can be the single source of truth for all users and groups. Configuring Entra ID as a provider allows for auto-discovery of user and group accounts, on-going synching of user data such as email address, name, and status, and integrated data mapping of field names and values.
+
+-   For instructions to configure your Entra ID system to integrate with authentik, refer to [Configure Entra ID](./setup-entra).
+-   For instructions to add Entra ID as a provider in authentik, refer to [Create a Entra ID provider](./add-entra-provider).
+
+## About using Entra ID with authentik
+
+The following sections discuss how Entra ID operates with authentik.
+
+### Discovery
+
+When first creating the provider and setting it up correctly, the provider will run a discovery and query your Entra ID for all users and groups, and attempt to match them with their respective counterparts in authentik. This discovery takes into consideration any **User filtering** options configured in the provider, such as only linking to authentik users in a specific group or excluding service accounts.
```

Comment:
"...correctly" is doing no work here, and implies the user's competence may not be suffcient.  

Suggestion: "When first creating and configuring the provider, authentik will run a discovery process and query your Entra ID..."